src/modules/75-cf/pages/feature-flags/FeatureFlagsPage.tsx

```diff
@@ -486,7 +486,7 @@ const FeatureFlagsPage: React.FC = () => {
 
   /* Hook needed because lazy loading being used on useGetAllFeatures above means changes to filters are NOT picked up when queryParams memo changes */
   useEffect(() => {
-    refetch({ queryParams: { ...queryParams } })
+    refetch({ queryParams: queryParams })
```

Comment:
Might as well simplify
```suggestion
    refetch({ queryParams })
```